.codeclimate.yml

```diff
@@ -1,8 +1,14 @@
 engines:
   fixme:
+    enabled: false
```

Comment:
p.s. I think engine-specific exclude_paths are in the works. 

This seems like an excellent use case!